packages/js-dpp/package.json

```diff
@@ -87,6 +87,7 @@
     "@apidevtools/json-schema-ref-parser": "^8.0.0",
     "@dashevo/dashcore-lib": "~0.19.28",
     "@dashevo/feature-flags-contract": "workspace:~",
+    "@dashevo/masternode-reward-shares-contract": "workspace:~",
```

Comment:
Don't we use it only for testing?

Reply:
We use `const masternodeRewardSharesTypes = require('@dashevo/masternode-reward-shares-contract/lib/rewardShareTypes');` in `getDataTriggersFactory`, like we did with`feature-flags-contract`

Reply:
Why we even need them? In case of feature flags it's important since we use them in Drive and it will be a lot of them. In this case, it doesn't matter. We just can use string.